apps/web/src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue

```diff
@@ -158,17 +159,16 @@ interface Props {
 
 const props = defineProps<Props>();
 
+const templateContainerRef = ref<HTMLElement | null>(null);
+const { arrivedState } = useScroll(templateContainerRef);
+const { bottom: isBottom, top: isTop } = toRefs(arrivedState);
 const state = reactive({
     selectedTemplateName: `${TEMPLATE_TYPE.DEFAULT}-${DASHBOARD_TEMPLATES.monthlyCostSummary.name}`,
```

Comment:
question: Should responsiveness be considered in this state?

Reply:
I think it was added due to the variable naming.

Reply:
+ When selecting a template, the name also changes.